automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookPyKernelSpec.scala

```diff
@@ -40,6 +40,14 @@ class NotebookPyKernelSpec extends ClusterFixtureSpec {
       }
     }
 
+    "should include Content-Security-Policy in headers" in { clusterFixture =>
+      withWebDriver { implicit driver =>
```

Comment:
Do we even need webDriver or a notebook for this? Couldn't the test just grab the headers from calling a Jupyter API directly?

Reply:
I was not sure how to create the URL for the notebook from just the project name and cluster name.

Reply:
With webDriver and notebook we can access the URL for the notebook directly

Reply:
That's what `notebooksBasePath(project, cluster)` does above. I think you could write this test as:
```
"should include Content-Security-Policy in headers" in { clusterFixture =>
  val headers = Leonardo.notebooks.getApiHeaders(clusterFixture.billingProject, clusterFixture.cluster)
  val contentSecurityHeader = headers.find(_.name == "Content-Security-Policy")
  contentSecurityHeader shouldBe 'defined
  contentSecurityHeader.get.value should include "https://bvdp-saturn-prod.appspot.com"
}
```
webDriver and notebook take up resources in the tests so best to avoid them when possible.